src/shared/components/footer/footer.css

```diff
@@ -0,0 +1,13 @@
+.footer-container,
+.footer-items {
+  justify-content: center;
+  align-items: center;
+  gap: 1rem;
+  font-size: 2rem;
```

Comment:
were these properties not present in component lib for footer?

Reply:
footer is not there my component library . As I don't need any footer over there .